ticketbuyer/buyer.go

```diff
@@ -199,61 +211,65 @@ func (t *TicketPurchaser) Purchase(height int64) (*PurchaseStats, error) {
 		ps.FeeMedian = info.FeeInfoBlocks[0].Median
 		ps.FeeMean = info.FeeInfoBlocks[0].Mean
 
-		// The expensive call to fetch all tickets in the mempool
-		// is here.
+		// Expensive call to fetch all tickets in the mempool
 		all, err := t.allTicketsInMempool()
 		if err != nil {
 			return ps, err
 		}
 		ps.MempoolAll = all
 	}
 
-	// Just starting up, initialize our purchaser and start
-	// buying. Set the start up regular transaction fee here
-	// too.
+	// Initialize based on where we are in the window
 	winSize := t.activeNet.StakeDiffWindowSize
+	thisIdxDiffPeriod := int(height % winSize)
+	nextIdxDiffPeriod := int(height + 1%winSize)
+	thisWindowPeriod := int(height / winSize)
+
+	refreshProportionLive := false
 	if t.firstStart {
-		t.idxDiffPeriod = int(height % winSize)
-		t.windowPeriod = int(height / winSize)
 		t.firstStart = false
-
-		log.Tracef("First run time, initialized idxDiffPeriod to %v",
-			t.idxDiffPeriod)
-
-		txFeeAmt, err := dcrutil.NewAmount(t.cfg.TxFee)
-		if err != nil {
-			log.Errorf("Failed to decode tx fee amount %v from config",
-				t.cfg.TxFee)
-		} else {
-			t.wallet.SetRelayFee(txFeeAmt)
```

Comment:
setting the txfee is no longer needed, as this was old workaround behavior for old defaults